source/agora/node/Validator.d

```diff
@@ -421,12 +421,7 @@ public class Validator : FullNode, API
     {
         return new Clock((out long time_offset)
             {
-                // not enrolled - no need to synchronize clocks
-                if (!this.enroll_man.isEnrolled(&this.utxo_set.peekUTXO))
-                    return false;
```

Comment:
Firstly because we need to know the block height to know if we are enrolled. That is not that easy before we have a ledger.
Secondly because I do not see any harm if we sync anyway.

Reply:
Own commit then